data/maps/zone1/act2/entities/rings.csv

```diff
@@ -0,0 +1,242 @@
+Adv2,RINGS,75,10
+7,1,29,19
```

Comment:
Couldn't use use the ring count enum for the ring box value?

Reply:
This is the regular rings, not itemboxes. :D
And I'm already using item macros for those, so I don't see a reason to change that (also my tokenizer is very dumb and doesn't really "get" enums).

Reply:
> This is the regular rings, not itemboxes. :D
🤦  oops yes of course